src/modules/did/did.service.ts

```diff
@@ -413,22 +415,42 @@ export class DIDService implements OnModuleInit {
 
         const token = await this.ipfsStore.get(serviceEndpoint);
 
-        const decodedData = jwt.decode(token) as {
-          claimData: Record<string, string>;
-        };
+        if (isJWT(token)) {
+          const decodedData = jwt.decode(token) as {
+            claimData: Record<string, string>;
+          };
 
-        if (!decodedData) {
-          return { serviceEndpoint, ...rest };
-        }
+          if (!decodedData) {
+            return { serviceEndpoint, ...rest };
+          }
+
+          const { claimData, ...claimRest } = decodedData;
 
-        const { claimData, ...claimRest } = decodedData;
+          return {
+            serviceEndpoint,
+            ...rest,
+            ...claimData,
+            ...claimRest,
+          };
+        }
```

Comment:
Do you think it makes sense to put this `isJWT` branch in the `try/catch` as well?